### DISCUSSION TITLE
Feature request: hide part of text

### DISCUSSION THREAD

**Discussion by francocalvo** [thegeeklab/hugo-geekdoc#361] [Ideas]
Right now I'm using Vimwiki to create my content. It allows me autogenerate a ToC inside the Markdown file which allows me to quickly jump between sections while writing.
Example:

```
# Contents

- [Descripción](#Descripción)
  - [Por hacer](#Descripción#Por hacer)
  - [Recursos](#Descripción#Recursos)
  - [Ejemplo de sub página](#Descripción#Ejemplo de sub página)
```

The problem is that Hugo can't render this as per [this post](https://discourse.gohugo.io/t/custom-markdown-links-stopped-working/24309). I'd like to use the ToC from geekdoc and hide the vimwiki one. 

Right now I'm using the expand shortcut, but it'd be nice if you could "delete" that from the final HTML.

**Comment by francocalvo** [comment]
Personally, I solved it by copying the `expand.html` in `layouts/shortcuts` and deleted all but the first line.

**Comment by xoxys** [comment]
Hi, this is a very specific issue that will not affect that many users, in my opinion. Nevertheless, you can handle it already using HTML e.g.:

```
<div class="hidden">
- [Descripción](#Descripción)
  - [Por hacer](#Descripción#Por hacer)
  - [Recursos](#Descripción#Recursos)
  - [Ejemplo de sub página](#Descripción#Ejemplo de sub página)
</div>
```
